docs/pages/accounts/account-types.md

```diff
@@ -2,48 +2,64 @@
+
+Personal Accounts are referred to as the default account type in the scenarios where you take action, but the activity does not specify which account it should belong.
 
-> This account is just for you, **never share access to this account for any reason.**
+> This account is just for you. Never share access to this account for any reason.
 
 ## Organizations
 
-If you are creating a project that may need to outlive your involvement with it, for instance a professional project for a company you work for, we recommend you create a new Organization for that project.
+When creating a project that needs to outlive your involvement with it, for instance, a professional project for a company you work for, we recommend you create a new Organization for that project.
 
-Common situations where Organizations are useful:
+Common situations where Organizations are helpful:
 
 - Transferring control of a project
 - Sharing access with others
 - Isolating expenses for work projects that you need to submit expenses for
-- Structuring projects for different contexts (e.g. work for different clients)
+- Structuring projects for different contexts, such as working for different clients
 
 |                                                                     | Personal Accounts | Organization |
 | ------------------------------------------------------------------- | ----------------- | ------------ |
-| **Create Projects**                                                 | ✅                 | ✅            |
-| **Build projects to submit to App Store and Play Store**            | ✅                 | ✅            |
-| **Release bug fixes with updates**                                  | ✅                 | ✅            |
-| **Transfer control of individual projects to another user**         | ✅                 | ✅
-| **Transfer control of all projects to another user**                |                   | ✅            |
-| **Programmatic access with limited privileges**                     |                   | ✅            |
-| **Designate multiple users who have complete control of a project** |                   | ✅            |
+| **Create Projects**                                                 | ✅                | ✅           |
+| **Build projects to submit to App Store and Play Store**            | ✅                | ✅           |
+| **Release bug fixes with updates**                                  | ✅                | ✅           |
+| **Transfer control of individual projects to another user**         | ✅                | ✅           |
+| **Transfer control of all projects to another user**                |                   | ✅           |
+| **Programmatic access with limited privileges**                     |                   | ✅           |
```

Comment:
I really like the idea of a table, however I don't think it works since the language around Project Transfers is misleading. Also Team Accounts is missing. There seems to be some misunderstanding of the differences between accounts. See my comment on line 13. We might just want to delete the table.